Automattic/blaze-ads
# Removes old dashboard compatibility classes

### What and why? 🤔

We are working on some tasks the DotOrg team left us to publish this plugin. 
This PR aims to fix a problem with incorrect usage of `<script>` tags inside our code. 

The affected class was a compatibility class that we no longer needed. We are removing those classes.

### Testing Steps ✍️

* Start your WordPress site with this version of the plugin active
* Verify that the Blaze Dashbord loads correctly

### Review checklist
